src/frontend/components/Filters/Filters.js

```diff
@@ -1,12 +1,32 @@
 import React, { useState, useEffect } from "react";
+import { useCategories, useFilters } from "../../contexts";
 import { filtersData } from "../../data";
 import { FilterCollection } from "../FilterCollection/FilterCollection";
-import { RadioInput } from "../RadioInput/RadioInput";
+import { Input } from "../Input/Input";
+import { RangeSlider } from "../RangeSlider/RangeSlider";
 import "./Filters.css";
+import {
+  CATEGORY,
+  CLEAR_FILTERS,
+  RATING,
+  SORT_BY,
+} from "./../../constants/filtersConstants";
 
 const Filters = () => {
   const [showResponsiveFilters, setShowResponsiveFilters] = useState(false);
   const { sortBys, ratings, collections } = filtersData;
+  const { categoryNames } = useCategories();
+  const { filters, dispatchFilters } = useFilters();
+  const {
+    // minPrice,
```

Comment:
Please remove the comments if they're not going to be used.

Reply:
I will se using it. Thank you though!